### DISCUSSION TITLE
Does this package support C#

### DISCUSSION THREAD

**Discussion by AQuarton** [XRM-OSS/D365-UI-Test#50] [Q&A]
Hi, I realise that it says typescript and js in the docs but any reason why this wouldn't support C# ?
Thanks

**Comment by bhavya999999** [comment]
An unopinionated UI testing library for Dynamics 365 CE and Dynamics Portals. Powered by TypeScript and playwright

**Comment by DigitalFlow** [comment]
HI @AQuarton,

to this date, C# is not supported.

There seems to be a C# version of playwright by now, but this package is based on the initial TypeScript version of playwright.

Kind regards,
Florian
